### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by AlexDernov**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,39 @@
...
+    <main>
+        <section class="question_box">
+            <button class="bookmark_corner"><img class="img_bookmarks" src="https://em-content.zobj.net/thumbs/240/htc/37/bookmark_1f516.png"/></button>
+            <h2 class="questions">What property flips the axes in flexbox?</h2>
+            <button class="show_answer">Show Answer</button>
+            <giv class="tags">
+                <buttom class="html">#html</buttom>
+                <buttom class="flexbox">#flexbox</buttom>
+                <buttom class="css">#css</buttom>
+            </giv>
```
Comment (Kheirah) [inline index.html]: Typo. You probably meant to say `div`.

File: index.html [hunk 2]
```diff
@@ -4,9 +4,39 @@
...
+        </section>
+        <section class="question_box">
+            <img/>
+            <h2 class="questions">Lorem Impsum dolor sit?</h2>
+            <button class="show_answer">Show Answer</button>
+            <giv class="tags">
+                <buttom class="html">#html</buttom>
+                <buttom class="flexbox">#flexbox</buttom>
+                <buttom class="css">#css</buttom>
+            </giv>
```
Comment (Kheirah) [inline index.html]: Same thing here :)

**Review by Kheirah — APPROVED** [review]
Overall really good!
Have a look at the typo though =)

File: index.html (hunk 1 above)
Comment (AlexDernov) [reply index.html]: > Typo. You probably meant to say `div`.

Thank`s! Typo is corrected.

**Pull request merged**